### ISSUE TITLE
Opening "Render" menu generates exception

### ISSUE THREAD

**Issue by follower** [MtnViewJohn/context-free#18]
When opening the "Render" menu (on Mac OS X 10.8.5 at least) the system log records the following exception:

```
[GView validateMenuItem:]: unrecognized selector sent to instance
```

It doesn't seem to prevent correct operation.

According to some posts I found (see below) these lines cause the exception (haven't looked at the code closely though, so it might just be the first one):

https://github.com/MtnViewJohn/context-free/blob/97b4074601a6726abd52340f27aff16f9ad3fa05/src-osx/GView.mm#L657

https://github.com/MtnViewJohn/context-free/blob/97b4074601a6726abd52340f27aff16f9ad3fa05/src-osx/CFDGDocument.mm#L551

See:

 * "[Falling through to super.validateMenuItem(:) generates Swift exception](https://stackoverflow.com/questions/49388800/falling-through-to-super-validatemenuitem-generates-swift-exception)"

[Note: This issue report is only a stub but hopefully it's better than no issue. :) ]

**Comment by MtnViewJohn** [comment]
I have updated GView to use validateUserInterfaceItem: and not call the superclass.

**Comment by MtnViewJohn** [comment]
Fixed in version 3.1
